ProcessMaker/Jobs/ImportProcess.php

```diff
@@ -517,6 +517,7 @@ protected function saveScreen($screen)
 
             return $new;
         } catch (\Exception $e) {
+            Log::info('*** Error: ' . $e->getMessage());
```

Comment:
Consider to change to `Log::error` to reflect this is an error during the import process.

Reply:
I agree, It was fixed